contentcuration/contentcuration/frontend/shared/views/StudioAccordion.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+
+  <div class="studio-accordion">
+    <div>
+      <KGrid
+        :style="{
+          backgroundColor: $themeTokens.surface,
+        }"
+      >
+        <KGridItem
+          :layout4="{ span: 4 }"
+          :layout8="{ span: 8 }"
+          :layout12="{ span: 12 }"
+        >
+          <h3>
+            <button
+              :id="id"
+              type="button"
+              :aria-expanded="showAccordionItem.toString()"
+              :aria-controls="id"
+              class="studio-accordion-header"
+              @click="openAccordion"
+            >
+              <span class="studio-accordion-title">
+                <slot name="left-actions"></slot>
+              </span>
+              <KIcon
+                :icon="showAccordionItem ? 'chevronUp' : 'chevronDown'"
+                :color="$themePalette.grey.v_400"
+                class="accordion-icon"
+              />
+            </button>
+          </h3>
+        </KGridItem>
+      </KGrid>
+    </div>
+
+    <KTransition kind="component-vertical-slide-out-in">
+      <div v-if="showAccordionItem">
+        <slot></slot>
+      </div>
+    </KTransition>
+    <div
+      v-if="!lastItem"
+      :style="{
+        borderBottom: `1.5px solid ${$themePalette.grey.v_200}`,
+      }"
+    ></div>
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'AccordionContainer',
+    props: {
+      lastItem: {
+        type: Boolean,
+        default: false,
+      },
+      id: {
+        type: String,
+        default: 'studio-accordion',
```

Comment:
Because this isn't set and is defaulted to `studio accordion`, the aria relationships break due to ID duplication